sink-connector/python/db_load/clickhouse_loader.py

```diff
@@ -495,7 +497,7 @@ def main():
     parser.add_argument('--dry_run', dest='dry_run',
                         action='store_true', default=False)
     parser.add_argument('--virtual_columns', help='virtual_columns',
-                        nargs='+', default=['`_sign`', '`_version`'])
+                        nargs='+', default=['`is_deleted`', '`_version`'])
```

Comment:
it does not hurt to add it, let us keep `_sign`

Reply:
indeed, if you use `is_deleted` you do not need `_sign`, will fix. If the sink-connector sees the column, it will populate it. 
those are the kind of decisions should be made obvious with a replication.yml file.

Reply:
https://github.com/Altinity/clickhouse-sink-connector/issues/313